src/api-reference/direct-connects/hotel-service-2/Introduction.markdown

```diff
@@ -6,89 +6,98 @@ layout: reference
 ## Disclaimer
```

Comment:
Add the pre-release include from the doc template, we'll add it to all the Hotel Service 2 pages. On this page, remove the Disclaimer section as the prerelease text covers it. 
{% include prerelease.html %}